geode-core/src/test/java/org/apache/geode/management/internal/cli/GfshParserParsingTest.java

```diff
@@ -362,4 +362,11 @@ public void testMultiLineCommand() throws Exception {
     assertThat(result.getParamValue("name")).isEqualTo("test");
     assertThat(result.getCommandName()).isEqualTo("start server");
   }
+
+  @Test
+  public void testShutdownWithOption() throws Exception {
+    String command = "shutdown --include-locators";
+    GfshParseResult result = parser.parse(command);
+    assertThat(result.getParamValue("include-locators")).isEqualTo("true");
```

Comment:
this line needs to change to the following to incorporate the changes on develop:

assertThat(result.getParamValue("include-locators")).isEqualTo(true);

Reply:
@jinmeiliao/reviewers,
What is the best practice when there are newer commits in develop which would cause things to break when the feature branch is merged into develop ? 

Should I be merging develop into my feature branch, make additional changes if necessary so that there are no breaking changes and only then submit a PR ? 
Which would mean that I should merge develop into my feature branch and then make the change as `assertThat(result.getParamValue("include-locators")).isEqualTo(true);` and then push the commits.

Reply:
Yes, that's the way I do my PRs: resolve any conflicts locally before pushing updates to the PR.

Reply:
Thanks @pdxrunner